resources.go

```diff
@@ -842,6 +893,18 @@ func Provider() tfbridge.ProviderInfo {
 					Source: "google_iam_policy.html.markdown",
 				},
 			},
+			"google_kms_crypto_key": {
+				Tok: gcpDataSource(gcpKMS, "getKMSCryptoKey"),
+				Docs: &tfbridge.DocInfo{
+					Source: "google_kms_crypto_key.html.markdown",
+				},
+			},
+			"google_kms_key_ring": {
+				Tok: gcpDataSource(gcpKMS, "getKMSKeyRing"),
+				Docs: &tfbridge.DocInfo{
+					Source: "google_kms_key_ring.html.markdown",
```

Comment:
If there are patterns here that are consistent but not being picked up automatically - we may want to embed theose patterns in the search heuristics used in `pulumi-terraform` so that we don't need as many manual annotations like this.  I'm not sure why this particular one doesn't get picked up automatically with the latest `pulumi-terraform` - I thought we added a rule to handle exactly this pattern.

Reply:
Not sure in this particular case, but I agree it's worth auditing what is being used and adding the various patterns where possible.